<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleHeatmapsView</class>
 <widget class="QWidget" name="MultipleHeatmapsView">
  <property name="geometry">
   <rect>
  <x>0</x>
  <y>0</y>
  <width>560</width>
  <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiple Heatmaps View</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
  <widget class="QGroupBox" name="groupBox">
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
      <width>40</width>
      <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Multiple Heatmaps view</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
      <width>40</width>
      <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="_scrol_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>528</width>
         <height>290</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="_image_grid"/>
        </item>
       </layout>
      </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="2">
       <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="_num_elem_lbl">
      <property name="text">
       <string># Elements: </string>
      </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="_num_sel_elem_lbl">
      <property name="text">
       <string># Selected elements: </string>
      </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
      <property name="text">
       <string># Columns</string>
      </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="_num_columns_sbox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Img Height</string>
      </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="_img_height_sbox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
       </widget>
      </item>
     </layout>
    </item>
     </layout>
    </item>
   </layout>
  </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
